spec/oga/xml/element_spec.rb

```diff
@@ -161,6 +165,22 @@
       @element.get('class').should == 'foo'
     end
 
+    it 'adds a new attribute' do
+      @element['class'] = 'foo'
+
+      @element['class'].should == 'foo'
+    end
+
+    it 'converts symbols to strings in names' do
+      @element[:foo] = 'foo'
+      @element[:foo].should == 'foo'
+      @element['foo'].should == 'foo'
+
+      @element['bar'] = 'bar'
+      @element[:bar].should == 'bar'
+      @element['bar'].should == 'bar'
+    end
```

Comment:
Also, this spec and the one for `Node#[]` can just check if `get` / `set` are called, they don't need to fully duplicate the tests just because they're aliases.